ci/report-exit.js

```diff
@@ -1,6 +1,16 @@
+const { generateDiscordCloudwatchLogFile, listECSTasks, sendDiscordNotification } = require('./helpers')
 const child_process = require('child_process')
 
-function main() {
+async function main() {
+  const taskArns = await listECSTasks()  // like  taskArns =  [ 'arn:aws:ecs:us-east-2:967314784947:task/ceramic-dev-tests/2466935a544f47ec9a1c3d8add235c84' ]
```

Comment:
We don't need the log urls in this message if they are already in the start messages. So I think most of this code (if not all?) can be removed.

Reply:
I disagree here, the exit failures will get duplicated into the separate failures channel (separate to results channel) therefore best if we duplicate the log entries here to avoid unnecessary flipping between channels.